### DISCUSSION TITLE
How can I increment a version number is a .csproj file?

### DISCUSSION THREAD

**Discussion by DavidThielen** [community/community#77446] [Actions] [answered]
### Select Topic Area

Question

### Body

In my csproj file (Blazor/C#) I have:

```xml
<Project Sdk="Microsoft.NET.Sdk.Razor">
    <PropertyGroup>
        <Version>2.2.38.3583</Version>
    </PropertyGroup>
```

How can I add to my action that, on a check-in, it increments the final number in the version? In this example it increments 3583 to 3584.

thanks - dave

**Comment by bodav** [comment, marked as answer]
You can try this action
https://github.com/marketplace/actions/net-project-version-updater

See scenario 3 for the build number bump

**Reply by DavidThielen** [reply]
This looks like it was written just for me - perfect.

thanks - dave
